### PR TITLE
fix(typings): Update type of Epic to add state$

### PR DESCRIPTION
**Pull request by cjol**
I had to add this to use it in my own project, so I figured I would share.

- [x] I have read the [Contributor Guide](https://github.com/redux-observable/redux-observable/blob/master/CONTRIBUTING.md)
- [x] My commit messages are in [conventional-changelog-standard](https://github.com/redux-observable/redux-observable/blob/master/CONTRIBUTING.md#sending-a-pull-request) format.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -26,8 +26,20 @@ export declare class ActionsObservable<T extends Action> extends Observable<T> {
...
+  value: S
+
+  // deprecated
+  getState(): S
+  dispatch: Store<S>["dispatch"]
+}
+
+
 export declare interface Epic<T extends Action, S, D = any, O extends T = T> {
-  (action$: ActionsObservable<T>, store: MiddlewareAPI<S>, dependencies: D): Observable<O>;
```
Comment (jayphelps) [inline index.d.ts]: Can you rename the parameter to state$?

**Comment by jayphelps** [comment]
Thank you!! Just the minor param rename and this looks good.

File: index.d.ts (hunk 1 above)
Comment (jayphelps) [reply index.d.ts]: I tried making this change myself but it looks like you don't have the option checked to "allow edits from maintainers". https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/

Comment (cjol) [reply index.d.ts]: Hm, that's strange - I think that option is enabled so I wonder if there is another issue. First pull request so I'm on unfamiliar ground!

Anyway, I think this is now fixed.

Comment (ajcrites) [reply index.d.ts]: This change doesn't remove the import of `MiddlewareAPI` which is no longer used.

**Pull request merged**